### PR TITLE
Resolve "todook"s after semgrep#2552 is resolved

### PR DESCRIPTION
**Pull request by IagoAbal**

### CODE REVIEW

**Comment by IagoAbal** [comment]
Semgrep wrapper seems to ignore these, but `semgrep-core -test_rules` expects matches in lines tagged with `todook`.

**Comment by mschwager** [comment]
Actually, it looks like https://github.com/returntocorp/semgrep/issues/2552 is fixed! So I think we can change these `todook` -> `ok` :+1:

File: javascript/lang/correctness/useless-assign.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ x3 = 1;
 x3 = 2;
 
 // https://github.com/returntocorp/semgrep/issues/2552
```
Comment (mschwager) [inline javascript/lang/correctness/useless-assign.js]:
```suggestion
```

File: javascript/lang/correctness/useless-assign.js [hunk 2]
```diff
@@ -11,12 +11,12 @@ x3 = 1;
...
-// todook:useless-assignment
+// todoruleid:useless-assignment
```
Comment (mschwager) [inline javascript/lang/correctness/useless-assign.js]:
```suggestion
// ok:useless-assignment
```

File: javascript/lang/correctness/useless-assign.js [hunk 3]
```diff
@@ -11,12 +11,12 @@ x3 = 1;
...
 x4 = {value1: 42};
 x4 = {x4, value2: 43};
 
 // https://github.com/returntocorp/semgrep/issues/2552
```
Comment (mschwager) [inline javascript/lang/correctness/useless-assign.js]:
```suggestion
```

File: javascript/lang/correctness/useless-assign.js [hunk 4]
```diff
@@ -11,12 +11,12 @@ x3 = 1;
...
-// todook:useless-assignment
+// todoruleid:useless-assignment
```
Comment (mschwager) [inline javascript/lang/correctness/useless-assign.js]:
```suggestion
// ok:useless-assignment
```

**Comment by IagoAbal** [comment]
Let me rebase this.

**Review by mschwager — APPROVED** [review]

**Comment by IagoAbal** [comment]
> Let me rebase this.

Too late... it's OK anyways.

**Pull request merged**